src/wallet/test/wallet_tests.cpp

```diff
@@ -86,7 +86,8 @@ BOOST_FIXTURE_TEST_CASE(scan_for_wallet_transactions, TestChain100Setup)
 {
     // Cap last block file size, and mine new block in a new block file.
     CBlockIndex* oldTip = m_node.chainman->ActiveChain().Tip();
-    GetBlockFileInfo(oldTip->GetBlockPos().nFile)->nSize = MAX_BLOCKFILE_SIZE;
+    const auto pos{WITH_LOCK(cs_main, return oldTip->GetBlockPos())};
+    GetBlockFileInfo(pos.nFile)->nSize = MAX_BLOCKFILE_SIZE;
```

Comment:
Same: Can include this in the cs_main scope as well, I think.

Reply:
done, thank you